src/test/lua/exasolvs/test_query_renderer.lua

```diff
@@ -1565,4 +1565,98 @@ function test_query_renderer.test_unknown_data_type_throws_error()
     assert_rendering_error_contains(original_query, "unknown data type")
 end
 
+function test_query_renderer.test_group_by_with_literal()
+    local original_query = {
+        type = "select",
+        aggregationType = "group_by",
+        selectList = { {type = "literal_exactnumeric", value = "1"} },
+        from = {type  = "table", name = "T1"},
+        groupBy = { {type = "literal_string", value = "a"} }
+    }
+    assert_renders_to(original_query, 'SELECT 1 FROM "T1" GROUP BY \'a\'')
+end
+
+function test_query_renderer.test_group_by_columns()
+    local original_query = {
+        type = "select",
+        aggregationType = "group_by",
+        selectList = {
+            {type = "column", name = "C1", tableName = "T1", columnNr = 0},
+            {type = "column", name = "C2", tableName = "T1", columnNr = 1}
+        },
+        from = {type  = "table", name = "T1"},
+        groupBy = {
+            {type = "column", name = "C1", tableName = "T1", columnNr = 0},
+            {type = "column", name = "C2", tableName = "T1", columnNr = 1}
+        }
+    }
+    assert_renders_to(original_query, 'SELECT "T1"."C1", "T1"."C2" FROM "T1" GROUP BY "T1"."C1", "T1"."C2"')
+end
+
+function test_query_renderer.test_group_by_columns()
```

Comment:
Copy paste error with same function name. Be careful, function overwriting is a legal operation in Lua, so you wont get a warning!

Reply:
It was also the same test. I've removed one